tests/drivers/sensor/sbs_gauge/src/test_smartbattery.c

```diff
@@ -0,0 +1,319 @@
+/*
+ * Copyright (c) 2021 Leica Geosystems AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <ztest.h>
+#include <drivers/sensor.h>
+
+static ZTEST_BMEM const struct device *dev;
+
+const struct device *get_fuel_gauge(void)
+{
+	return device_get_binding(DT_LABEL(DT_ALIAS(bat0)));
+}
```

Comment:
```suggestion
static const struct device *dev = DEVICE_DT_GET_ANY(sbs_gauge);
```

Then check it with `device_is_ready()` before first use.